src/Assets/icons/Logo.jsx

```diff
@@ -2,7 +2,7 @@ import { useTheme } from "../../context/theme-context"
 export const Logo = ({width,height})=>{
     const {theme} = useTheme();
     return(
-        <svg width={width} height={height} viewBox="0 0 512 512" color={theme==="dark" ? "#d6806a":"#ca2535"}>
+        <svg width={width} height={height} viewBox="0 0 512 512" color="#d6806a">
```

Comment:
Inline styling is not a good idea to go with. Try accessing styling using class

Reply:
For icons I am taking height and width props and then setting the height width attributes accordingly